app/controllers/vendor_api/decisions_controller.rb

```diff
@@ -78,5 +119,16 @@ def render_validation_errors(errors)
       error_responses = errors.full_messages.map { |message| { error: 'ValidationError', message: message } }
       render status: :unprocessable_entity, json: { errors: error_responses }
     end
+
+    def render_cannot_find_course_option
```

Comment:
It's a shame not to use the existing `render_validation_errors` method. Could we pass `error`s down from the form?